.github/workflows/tests.yml

```diff
@@ -28,14 +28,14 @@ jobs:
         # Display the Python version being used
       - name: Display Python version
         run: python -c "import sys; print(sys.version)"
-        # Install the package using the setup.py
-      - name: Install package
-        run: python setup.py install
         # Install pytest (you can use some other testing utility)
       - name: Install deps
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt -r test-requirements.txt
         # Run the tests. I'm using pytest and the file is in the tests directory.
+        # Install the package using the setup.py
```

Comment:
Well, that's just not true anymore. First of all, it's used for building and not installing (`pip install` would just unzip the wheel for installing). And second, it's being deleted :man_shrugging: